.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: python
+python:
+  - "2.7"
+
+cache: pip
+
+notifications:
+  email: false
+
+git:
+  depth: 3
+
+services:
+  - docker
+
+env:
+  - TEST_ARG="make test-documentation"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.Experimental"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.Fluid.{Actuators,BaseClasses,Chillers,Delays,Examples,FMI,FixedResistances}"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.Fluid.{HeatExchangers,HeatPumps,Interfaces,MassExchangers,MixingVolumes,Movers,Sensors,Sources,Storage}"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.ThermalZones"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.{Airflow,BoundaryConditions,Controls}"
+  - TEST_ARG="make test-dymola PACKAGE=IBPSA.{Media,Utilities}"
+
+before_install:
+  - sudo mv IBPSA/Resources/Scripts/travis/usr/local/bin/dymola /usr/local/bin/
+  - sudo chmod +x /usr/local/bin/dymola
+  - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+  - docker pull "$DOCKER_USERNAME"/travis_ubuntu-1604_dymola-2017fd01-x86-64
+
+# Install dependencies
+# For sphinx, we need to install specific package versions, otherwise
```

Comment:
Lines 32-34 can probably be deleted, or will there be a Sphinx based documentation later?